frontend/public/components/modals/labels-modal.tsx

```diff
@@ -35,9 +36,21 @@ const BaseLabelsModal: React.FC<BaseLabelsModalProps> = ({
   const [labels, setLabels] = React.useState(
     SelectorInput.arrayify(_.get(resource, path.split('/').slice(1))),
   );
+  const [watchedResource, watchedResourceLoaded] = useK8sWatchResource<K8sResourceCommon>({
+    kind: resource.kind,
+    name: resource?.metadata?.name,
+    namespace: resource?.metadata?.namespace,
+  });
```

Comment:
If we use optional chaining for the other properties, we probably need it here too:
```suggestion
  const [watchedResource, watchedResourceLoaded] = useK8sWatchResource<K8sResourceCommon>({
    kind: resource?.kind,
    name: resource?.metadata?.name,
    namespace: resource?.metadata?.namespace,
  });
```

Reply:
We have instances all over the code base there `kind` and `metadata` don't use optional chaining.  But now the linter complains about `metadata` but not `kind`.  Wonder what changed?  It wasn't https://github.com/openshift/console/blame/af5b08f9be05e01333b4595c11006657fe952ceb/frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts#L60-L61